.travis.yml

```diff
@@ -0,0 +1,7 @@
+language: python
+python:
+  - "3.5"
```

Comment:
include 3.6 and 3.7 so we cover default python versions on xenial/bionic/focal

Reply:
done